github-archive/launch.sh

```diff
@@ -12,13 +12,33 @@ if [[ -z "$2" ]] ; then
   exit 1
 fi
 
+function sed_replace() {
```

Comment:
sed under linux usually has a `-i` parameter which causes it to edit files in place. Is that missing in the OS X version of sed or is there something else this method does that I'm missing?